spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -234,22 +234,15 @@
       allow(user).to receive(:establishing_in_person_enrollment).and_return(enrollment)
     end
 
-    it 'sets ssn and state_id_type on pii_from_user' do
+    it 'sets ssn on pii_from_user' do
       expect(Idv::Agent).to receive(:new).with(
         hash_including(
```

Comment:
(suggestion, non-blocking): This is more of a regression testing thing, but it would be good to have an expectation that ensures we aren't sending `state_id_type` as `hash_including` cannot assert the absence of a key-value pair.

Reply:
This is a weird case and I'm on the fence. My approach with this was that it is no longer reasonable to require `state_id_type`'s presence, but I'm not sure I'd go as far as requiring its absence here. "The controller no longer overwrites the value with a lie" feels like a kind of weird regression test after I ripped that code out.